test/unit/repo_test.rb

```diff
@@ -126,4 +126,44 @@ class RepoTest < ActiveSupport::TestCase
     assert_not repos.include?(subscribed_repo)
     assert repos.include?(unsubscribed_repo)
   end
+
+  # test '.order_by_issue_count' do
+  #  expected_order_by_issues = [repos(:issue_triage_sandbox), repos(:rails_rails)]
+
+    # repos = Repo.order_by_issue_count
+
+    # assert_equal repos, expected_order_by_issues
+  # end
```

Comment:
Tried adding some tests for other related queries as well. I faced some complications because the database already have existing data. Would it be reasonable to add a gem like [DatabaseCleaner](https://github.com/DatabaseCleaner/database_cleaner) to perform a teardown between tests?